setup.py

```diff
@@ -35,14 +35,11 @@
         'dev': [
             "coverage==4.2",
             "flake8==3.2.1",
-            "mccabe==0.6.1",
-            "pycodestyle==2.3.1",
-            "pyflakes==1.5.0",
             "pylint==1.7.1",
             "pytest==2.7.3",
             "pytest-cov==2.1.0",
             "pytest-timeout==0.4",
-            "urllib3==1.10.2",
+            "urllib3==1.21.1",
```

Comment:
ah, of course. i think i still like the idea of pinning everything in `dev`. since it's dev, there's no problem bumping these and i think we tend to try to keep them recent when we remember.

(i thought requests bundled its own urllib3 but as above happy to bump)